code/modules/client/verbs/suicide.dm

```diff
@@ -256,6 +256,9 @@
 		message_admins("[key_name(src)] (job: [src.job ? "[src.job]" : "None"]) committed suicide at [AREACOORD(src)].")
 
 /mob/living/proc/canSuicide()
+	var/area/A = get_area(src)
+	if(A.area_flags & BLOCK_SUICIDE)
+		to_chat(src, "<span class='warning'>You can't commit suicide here! You can ghost if you'd like.</span>")
```

Comment:
this doesn't actually return, meaning it won't do anything

Reply:
guh

Reply:
fixed